ckanext/datastore/logic/action.py

```diff
@@ -456,9 +456,9 @@ def datastore_records_delete(context: Context, data_dict: dict[str, Any]):
     :rtype: dictionary
 
     '''
+    filters = data_dict.pop('filters', None)
```

Comment:
This is super minor, but I used `get` in my suggested change to save an unnecessary modification to the dict here.

Reply:
A right, sorry, makes sense as we are really not doing anything else with the data_dict in this method. Okay should be updated now!